content/hi/_index.md

```diff
@@ -17,6 +17,8 @@ title: क्लाउड नेटिव शब्दावली
 क्लाउड नेटिव शब्दावली की शुरुआत CNCF मार्केटिंग समिति(CNCF Marketing Committee, BVS) द्वारा की गई थी और इसमें [Catherine Paganini](https://www.linkedin.com/in/catherinepaganini/en/), [Chris Aniszczyk](https://www.linkedin.com/in/caniszczyk/),
 [Daniel Jones](https://www.linkedin.com/in/danieljoneseb/?originalSubdomain=uk), [Jason Morgan](https://www.linkedin.com/in/jasonmorgan2/), [Katelin Ramer](https://www.linkedin.com/in/katelinramer/), [Mike Foster](https://www.linkedin.com/in/mfosterche/?originalSubdomain=ca), [Seokho Son](https://www.linkedin.com/in/seokho-son/) और कई अन्य योगदानकर्ताओं के योगदान शामिल हैं।
 
+क्लाउड नेटिव शब्दावली का हिंदी स्थानीयकरण [Hindi localization team](https://cloud-native.slack.com/archives/C02PCHEQXK6) द्वारा शुरू किया गया था और इसमें [Anubhav Vardhan](https://www.linkedin.com/in/anubhav-vardhan-a89587192/), [Garima Negi](https://www.linkedin.com/in/garima-negi-03980ba8/), [Jayesh Srivastav](https://www.linkedin.com/in/jayesh-srivastava/), [Sayantani Saha](https://www.linkedin.com/in/sayantani-saha-47a55b1bb/), [Kunal Verma](https://www.linkedin.com/in/verma-kunal/), [Aayush Sharma](https://www.linkedin.com/in/aayush-sharma967/), [Anurag Kumar](https://www.linkedin.com/in/placeholder/), [Shivam Tyagi](https://www.linkedin.com/in/shivam-tyagi-57b7341a6/?originalSubdomain=in), [Shubham Chaudhari](https://www.linkedin.com/in/shubham28698/) और कई अन्य योगदानकर्ताओं के योगदान शामिल हैं।
```

Comment:
nit: it will be okay to remove this, because it seems that this makes no difference, and for consistency among the LinkedIn URLs of Hindi contributors.

```suggestion
क्लाउड नेटिव शब्दावली का हिंदी स्थानीयकरण [Hindi localization team](https://cloud-native.slack.com/archives/C02PCHEQXK6) द्वारा शुरू किया गया था और इसमें [Anubhav Vardhan](https://www.linkedin.com/in/anubhav-vardhan-a89587192/), [Garima Negi](https://www.linkedin.com/in/garima-negi-03980ba8/), [Jayesh Srivastav](https://www.linkedin.com/in/jayesh-srivastava/), [Sayantani Saha](https://www.linkedin.com/in/sayantani-saha-47a55b1bb/), [Kunal Verma](https://www.linkedin.com/in/verma-kunal/), [Aayush Sharma](https://www.linkedin.com/in/aayush-sharma967/), [Anurag Kumar](https://www.linkedin.com/in/placeholder/), [Shivam Tyagi](https://www.linkedin.com/in/shivam-tyagi-57b7341a6/), [Shubham Chaudhari](https://www.linkedin.com/in/shubham28698/) और कई अन्य योगदानकर्ताओं के योगदान शामिल हैं।
```